pym/bob/input.py

```diff
@@ -1261,6 +1263,7 @@ def __init__(self, corePackage, checkout=None, checkoutSCMs=[],
             isValid = False
             self.scmList = []
 
+        self.checkoutAsserts = [ CheckoutAssert (a, fullEnv) for a in corePackage.recipe.checkoutAsserts ]
```

Comment:
Variable should probably be private.

```suggestion
        self.__checkoutAsserts = [ CheckoutAssert (a, fullEnv) for a in corePackage.recipe.checkoutAsserts ]
```